src/modules/billing/services/two-part-tariff-service/index.js

```diff
@@ -0,0 +1,91 @@
+const { set } = require('lodash');
+const twoPartTariffMatching = require('./two-part-tariff-matching');
+const returnHelpers = require('./returns-helpers');
+const Purpose = require('../../../../lib/models/purpose');
+
+const mapResultsWithData = (overallError, data, transactions) => {
+  const updatedTransactions = [];
```

Comment:
I don't think this array is doing anything, I think if the function returned `transactions` at the end it may have the same effect?|

Reply:
You're right. I thought I was receiving a copy of the transaction from filter, but I guess not. I should've double checked this, but I will know for next time!